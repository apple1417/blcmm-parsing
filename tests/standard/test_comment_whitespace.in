<BLCMM v="1">
#<!!!You opened a file saved with BLCMM in FilterTool. Please update to BLCMM to properly open this file!!!>
	<head>
		<type name="BL2" offline="false"/>
		<profiles>
			<profile name="default" current="true"/>
		</profiles>
	</head>
	<body>
		<category name="root">
			<comment>   	  	   </comment>
			<comment>	</comment>
			<comment>     		  	 	</comment>
			<comment>	</comment>
			<comment>     		 		  </comment>
			<comment>	</comment>
			<comment>     		 		  </comment>
			<comment>	</comment>
			<comment>     		 				</comment>
			<comment>	</comment>
			<comment>     	 		  </comment>
			<comment>	</comment>
			<comment>     	     </comment>
			<comment>	</comment>
			<comment>     			 			</comment>
			<comment>	</comment>
			<comment>     		 				</comment>
			<comment>	</comment>
			<comment>     			  	 </comment>
			<comment>	</comment>
			<comment>     		 		  </comment>
			<comment>	</comment>
			<comment>     		  	  </comment>
			<comment>	</comment>
			<comment>     	    	</comment>
			<comment>	</comment>
			<comment>  </comment>
			<comment></comment>
			<comment></comment>
		</category>
	</body>
</BLCMM>

#Commands:

