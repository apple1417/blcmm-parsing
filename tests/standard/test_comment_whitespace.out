<BLCMM v="1">
	<head>
		<type name="BL2" offline="false"/>
		<profiles>
			<profile name="default" current="true"/>
		</profiles>
	</head>
	<body>
		<category name="root">
			<comment>   	  	   </comment>
			<comment>	</comment>
			<comment>     		  	 	</comment>
			<comment>	</comment>
			<comment>     		 		  </comment>
			<comment>	</comment>
			<comment>     		 		  </comment>
			<comment>	</comment>
			<comment>     		 				</comment>
			<comment>	</comment>
			<comment>     	 		  </comment>
			<comment>	</comment>
			<comment>     	     </comment>
			<comment>	</comment>
			<comment>     			 			</comment>
			<comment>	</comment>
			<comment>     		 				</comment>
			<comment>	</comment>
			<comment>     			  	 </comment>
			<comment>	</comment>
			<comment>     		 		  </comment>
			<comment>	</comment>
			<comment>     		  	  </comment>
			<comment>	</comment>
			<comment>     	    	</comment>
			<comment>	</comment>
			<comment>  </comment>
			<comment></comment>
			<comment></comment>
		</category>
	</body>
</BLCMM>
